astropy/io/ascii/ecsv.py

```diff
@@ -17,8 +17,6 @@
 from astropy.utils.exceptions import AstropyUserWarning, AstropyWarning
```

Comment:
lint: because of the change below, `AstropyWarning` is no longer used in this module, so the import can be removed (`AstropyUserWarning`, however, is still used elsewhere in the file)

Reply:
Done! what linter do you use? I have mine turned off when working on Astropy, because they all have fits about import order and line lengths, etc.

Reply:
With emacs, it just picks up the configuration from `setup.cfg`, so I get red squiggly lines under anything not OK.